src/client/components/ProductsContainer.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ProductsList from './ProductsList';
+
+const ProductsContainer = ({ products, isArchive }) => (
+  <div className="products">
+    <header className="products__header">
+      <h2 className="products__heading">
+        {isArchive ? 'Orders history' : 'Products list'}
+      </h2>
+      <span className="products__info">
+        {isArchive ? 'Remove item' : 'Mark as ordered'}
+      </span>
+    </header>
+
+    <ProductsList isArchive={isArchive} products={products} />
+  </div>
+);
+
+ProductsContainer.defaultProps = {
+  isArchive: false
```

Comment:
do we need this? this is falsy by default

Reply:
Eslint forces to set default props when none was passed down

Reply:
I've changed this Eslint rule to false